### PR TITLE
Ui grid view

### PR DESCRIPTION
**Pull request by almoral**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,22 +22,29 @@
     "@angular/platform-browser-dynamic": "^5.2.0",
     "@angular/router": "^5.2.0",
     "@nicky-lenaers/ngx-scroll-to": "^0.6.0",
+    "@types/fullcalendar": "^3.8.0",
     "angular-in-memory-web-api": "^0.5.1",
     "angular2-materialize": "^15.1.10",
+    "angular4-datepicker": "^1.0.1",
```
Comment (fiallega) [inline package.json]: is there not something called angular5-datepicker ?? the fact that it has angular4 is already a redflag for me to think about. Just raising that flag

File: package.json [hunk 2]
```diff
@@ -46,7 +53,7 @@
     "@angular/language-service": "^5.2.0",
     "@types/jasmine": "^2.5.53",
     "@types/jasminewd2": "^2.0.2",
-    "@types/jquery": "^3.2.16",
+    "@types/jquery": "^3.2.7",
```
Comment (fiallega) [inline package.json]: downgrading jquery .... reason ?

Comment (fiallega) [inline package.json]: there are four libraries related to calendar: fullcalendar, angular4-datepicker, mydatepicker, and ng-fullcalendar". Are they all needed ?

File: package.json (hunk 1 above)
Comment (almoral) [reply package.json]: No. I would have to ask them why they named it that.

File: package.json (hunk 2 above)
Comment (almoral) [reply package.json]: The date picker uses that version.

Comment (almoral) [reply package.json]: mydatepicker is the actual date picker we are using. I'll remove the angular4-datepicker. That was the old version.

File: package.json [hunk 3]
```diff
@@ -22,22 +22,28 @@
     "@angular/platform-browser-dynamic": "^5.2.0",
     "@angular/router": "^5.2.0",
     "@nicky-lenaers/ngx-scroll-to": "^0.6.0",
+    "@types/fullcalendar": "^3.8.0",
```
Comment (fiallega) [inline package.json]: does @types/fullcalendar shouldn't it go in devDependcies like the rest of them. Consult with Ramon, he understands @types very well

File: src/app/calendar-grid-view/calendar-grid-view.component.ts [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+
+
+@Component({
+  encapsulation: ViewEncapsulation.None,
+  selector: 'mdc-calendar-grid-view',
+  templateUrl: './calendar-grid-view.component.html',
+  styleUrls: ['./calendar-grid-view.component.css']
+})
+export class CalendarGridViewComponent implements OnInit {
+
```
Comment (fiallega) [inline src/app/calendar-grid-view/calendar-grid-view.component.ts]: I dont think 'View' in 'CalendarGridViewComponent' is necessary. It is a grid component. (Components have views associated with it)

File: package.json (hunk 3 above)
Comment (almoral) [reply package.json]: Alright, I'll discuss with him. I was following their instructions for installing.

**Pull request merged**